void-files/etc/firewall-conf/open-out.conf

```
@@ -15,5 +15,7 @@ add rule inet filter input iif lo accept
 add rule inet filter input ct state established,related accept
 # IPv6 neighbor discovery
 add rule inet filter input ip6 nexthdr icmpv6 icmpv6 type { nd-neighbor-solicit, echo-request, nd-router-advert, nd-neighbor-advert } accept
+# include file with custum inbound rules
+include custom-inbound.conf
```

Make this work for any rules tagged "custom" in the firewall-conf directory.
```suggestion
include custom-*.conf
```